### PR TITLE
Task#218543: Remove baseUrl from all api's

### PR DESCRIPTION
**Pull request by snehalsabade**

### CODE REVIEW

File: packages/nulp_elite/src/pages/search/DomainList.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ const DomainList = () => {
       setIsLoading(false);
     }
     try {
-      const url = `http://localhost:3000/api/framework/v1/read/nulp?categories=board,gradeLevel,medium,class,subject`;
+      const url = `/api/framework/v1/read/nulp?categories=board,gradeLevel,medium,class,subject`;
```
Comment (komalm) [inline packages/nulp_elite/src/pages/search/DomainList.js:104]: @snehalsabade framework is also hardcoded?
cc @paritshivani

File: packages/nulp_elite/src/pages/content/CategoryPage.js [hunk 2]
```diff
@@ -120,7 +120,7 @@ const CategoryPage = () => {
     } finally {
     }
     try {
-      const url = `http://localhost:3000/api/framework/v1/read/nulp?categories=board,gradeLevel,medium,class,subject`;
+      const url = `/api/framework/v1/read/nulp?categories=board,gradeLevel,medium,class,subject`;
```
Comment (komalm) [inline packages/nulp_elite/src/pages/content/CategoryPage.js:123]: @snehalsabade Framework is hardcoded cc @paritshivani

File: packages/nulp_elite/src/pages/search/DomainList.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ const DomainList = () => {
       Cookie: `connect.sid=${getCookieValue("connect.sid")}`,
     };
     try {
-      const url = `http://localhost:3000/api/channel/v1/read/0130701891041689600`;
+      const url = `/api/channel/v1/read/0130701891041689600`;
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply packages/nulp_elite/src/pages/search/DomainList.js:93]: Need to store rootOrgId in zustand store
-( working on it in other PR)

File: packages/nulp_elite/src/pages/content/CategoryPage.js (hunk 2 above)
Comment (snehalsabade) [reply packages/nulp_elite/src/pages/content/CategoryPage.js:123]: framework - Need to store in zustand store

File: packages/nulp_elite/src/pages/content/CategoryPage.js [hunk 4]
```diff
@@ -110,7 +110,7 @@ const CategoryPage = () => {
       Cookie: `connect.sid=${getCookieValue("connect.sid")}`,
     };
     try {
-      const url = `http://localhost:3000/api/channel/v1/read/0130701891041689600`;
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply packages/nulp_elite/src/pages/content/CategoryPage.js:113]: Need to store rootOrgId in zustand store

**Pull request merged**